service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/Order.java

```diff
@@ -264,14 +264,6 @@
         IsFinalOrder.NO,
         Constants.MANUAL_UPLOAD_QUESTIONS
     ),
-    C43_CHILD_ARRANGEMENTS_SPECIFIC_ISSUE_PROHIBITED_STEPS_ORDER(
```

Comment:
wasn't removed as a part of the other story?

Reply:
Looks like I didn't spot this in that story due to a minor typo, I will correct it in that story as well.

My other story had C43_CHILD_ARRANGEMENT_SPECIFIC_ISSUE_PROHIBITED_STEPS_ORDER which is missing an "s" on arrangement